internal/template-validator/webhooks/utils.go

```diff
@@ -69,29 +70,35 @@ func ToAdmissionResponse(causes []metav1.StatusCause) *admissionv1.AdmissionResp
 	}
 }
 
-func GetAdmissionReviewVM(ar *admissionv1.AdmissionReview) (*kubevirt.VirtualMachine, *kubevirt.VirtualMachine, error) {
+func GetAdmissionReviewVM(ar *admissionv1.AdmissionReview) (*kubevirt.VirtualMachine, error) {
 	if ar.Request.Resource.Resource != "virtualmachines" {
-		return nil, nil, fmt.Errorf("expect resource %v to be '%s'", ar.Request.Resource, "virtualmachines")
+		return nil, fmt.Errorf("expect resource %v to be '%s'", ar.Request.Resource, "virtualmachines")
 	}
 
-	var err error
-	raw := ar.Request.Object.Raw
-	newVM := kubevirt.VirtualMachine{}
-
-	err = json.Unmarshal(raw, &newVM)
+	newVM := &kubevirt.VirtualMachine{}
+	err := json.Unmarshal(ar.Request.Object.Raw, newVM)
 	if err != nil {
-		return nil, nil, err
+		return nil, err
```

Comment:
you can use directly
`return vm, err`

Reply:
`newVM` is not `nil` in case of error. But maybe it does not matter.

Reply:
I think it doesn't matter, because if err != nil then it will fail in fucntion which called the GetAdmissionReviewVM function